source/particle/interpolator/bilinear_least_squares.cc

```diff
@@ -290,9 +364,14 @@ namespace aspect
     {
       ASPECT_REGISTER_PARTICLE_INTERPOLATOR(BilinearLeastSquares,
                                             "bilinear least squares",
-                                            "Interpolates particle properties onto a vector of points using a "
-                                            "bilinear least squares method. "
-                                            "Note that deal.II must be configured with BLAS and LAPACK to "
+                                            "Uses linear least squares to obtain the slopes and center of a 2D or "
+                                            "3D plane from the particle positions and a particular property value "
+                                            "on those particles. "
+                                            "Interpolate this property onto the support points or to initiate the "
+                                            "property value on a new particle. If the limiter is enabled then it "
```

Comment:
Avoid descriptions of how this is used at the moment, in the future this might be changed. This function only knows that it interpolates properties onto new locations, what these locations are (integration points, new particles, postprocessing locations) are not known to the plugin.
```suggestion
                                            "Interpolate this property onto a vector of points. "
                                            "If the limiter is enabled then it "
```